### PR TITLE
combinatives weight for TOPSIS

### PR DESCRIPTION
**Pull request by storyofhis**

### CODE REVIEW

File: cmd/database/migrations/up/main.go [hunk 1]
```diff
@@ -176,6 +176,19 @@ BEFORE UPDATE ON expectations
...
+CREATE TABLE "topsis_combinative_to_alternatives" (
+  "id" BIGSERIAL PRIMARY KEY,
+  "score" decimal,
+  "topsis_id" bigint,
+  "alternative_id" int,
+  "created_at" TIMESTAMP NOT NULL DEFAULT (now()),
+  "updated_at" TIMESTAMP NOT NULL DEFAULT (now())
+);
+CREATE TRIGGER set_topsis_combinative_to_alternatives_timestamp
+BEFORE UPDATE ON expectations
```
Comment (albugowy15) [inline cmd/database/migrations/up/main.go]: bukan `expectations`, tapi `topsis_combinative_to_alternatives`

File: internal/repositories/recommendations.go [hunk 2]
```diff
@@ -121,6 +121,23 @@ func GetTOPSISAHPRecommendations(studentId string) ([]models.RecommendationResul
...
+		INNER JOIN alternatives a ON a.id = ata.alternative_id
+		INNER JOIN topsis ON topsis.id = ata.topsis_id
+		WHERE topsis.student_id = $1
+		ORDER BY ata.score DESC`,
+		studentId,
+	)
+	if err != nil {
+		log.Println("db err : ", err)
+	}
+	return recommendations, nil
```
Comment (albugowy15) [inline internal/repositories/recommendations.go]: return `err` nya

**Review by albugowy15 — CHANGES_REQUESTED** [review]

**Comment by albugowy15** [comment]
di `migrations/down/main.go` tambahin sql buat drop table `topsis_combinative_to_alternatives`

File: cmd/database/migrations/up/main.go (hunk 1 above)
Comment (albugowy15) [reply cmd/database/migrations/up/main.go]: sama juga buat `topsis_ahp_to_alternatives`

Comment (storyofhis) [reply cmd/database/migrations/up/main.go]: DONE!!

File: internal/repositories/recommendations.go (hunk 2 above)
Comment (storyofhis) [reply internal/repositories/recommendations.go]: DONE!!

File: cmd/database/migrations/up/main.go [hunk 3]
```diff
@@ -172,7 +172,7 @@ CREATE TABLE "topsis_ahp_to_alternatives" (
   "updated_at" TIMESTAMP NOT NULL DEFAULT (now())
 );
 CREATE TRIGGER set_topsis_ahp_to_alternatives_timestamp
-BEFORE UPDATE ON expectations
+BEFORE UPDATE ON topsis_combinative_to_alternatives
```
Comment (albugowy15) [inline cmd/database/migrations/up/main.go]: yang bener `topsis_ahp_to_alternatives`

**Review by albugowy15 — CHANGES_REQUESTED** [review]

**Review by albugowy15 — APPROVED** [review]

**Pull request merged**